TBufferMergerRootOutputer.cc

```diff
@@ -41,6 +42,9 @@ TBufferMergerRootOutputer::TBufferMergerRootOutputer(std::string const& iFileNam
                     autoFlush_{iConfig.autoFlush_ != -1 ? iConfig.autoFlush_ : Config::kDefaultAutoFlush },
                     concurrentWrite_{iConfig.concurrentWrite}
 {
+   if(iConfig.cacheSize_ > 0 ) {
+      new TFileCacheWrite(buffer_.GetFile()->CurrentFile(), iConfig.cacheSize_);
+   }
```

Comment:
@pcanal is this what is needed?

Reply:
Actually no, if I read correctly, that file is the `TMemFile` so the Write cache is useless.